fastlane_core/spec/project_spec.rb

```diff
@@ -299,7 +299,12 @@ def within_a_temp_dir
       it "SUPPORTED_PLATFORMS should be iphonesimulator iphoneos" do
         options = { project: "./fastlane_core/spec/fixtures/projects/Example.xcodeproj" }
         @project = FastlaneCore::Project.new(options, xcodebuild_list_silent: true, xcodebuild_suppress_stderr: true)
-        expect(FastlaneCore::Project).to receive(:run_command).with("xcodebuild clean -showBuildSettings -project ./fastlane_core/spec/fixtures/projects/Example.xcodeproj 2> /dev/null", { timeout: 10, retries: 3, print: false }).and_return(File.read("./fastlane_core/spec/fixtures/projects/build_settings_with_toolchains"))
+        if FastlaneCore::Helper.xcode_higher_than_8_3?
```

Comment:
i think this should be splitted up into 2 tests. with mocked return of `xcode_atleast` on time with `false` and one time with `true`  to be sure that it works on old and new xcodes.

the current variation is depending on the local xcode installation.